### PR TITLE
Update fyber marketplace 7.4.1

### PR DESCRIPTION
**Pull request by AsafKoskiWork**
- updated used FMP version to 7.4.1
- Add a null check for completion callback. Potential crash fix
- Convert app ids collection to set, so the warning message will only appear if we have multiple unique app ids

### CODE REVIEW

File: ThirdPartyAdapters/fyber/fyber/src/main/java/com/google/ads/mediation/fyber/FyberMediationAdapter.java [hunk 1]
```diff
@@ -158,12 +160,11 @@ public void initialize(Context context,
...
 
         // We can only use one app id.
-        String appIdForInitialization = configuredAppIds.get(0);
-
+        String appIdForInitialization = configuredAppIds.iterator().next();
         if (configuredAppIds.size() > 1) {
             String message = String.format("Multiple '%s' entries found: %s. " +
                             "Using '%s' to initialize the Fyber Marketplace SDK.",
-                    KEY_APP_ID, appIdForInitialization, appIdForInitialization);
+                    KEY_APP_ID, configuredAppIds, appIdForInitialization);
```
Comment (spilaka) [inline ThirdPartyAdapters/fyber/fyber/src/main/java/com/google/ads/mediation/fyber/FyberMediationAdapter.java]: ConfiguredAppIds returns instances of set object some thing like this '@61bbe9ba'. So, I think it should be appIdForInitialization.

File: ThirdPartyAdapters/fyber/fyber/src/main/java/com/google/ads/mediation/fyber/FyberMediationAdapter.java [hunk 2]
```diff
@@ -178,21 +178,23 @@ public void initialize(Context context,
     private void waitForInitializationStatusAndReport(
             final @NonNull InitializationCompleteCallback completionCallback) {
```
[the remark this answers is not in the document]
Comment (not-there-yet) [reply ThirdPartyAdapters/fyber/fyber/src/main/java/com/google/ads/mediation/fyber/FyberMediationAdapter.java]: Apparently it is necessary, please see #213

[the remark this answers is not in the document]
Comment (not-there-yet) [reply ThirdPartyAdapters/fyber/fyber/src/main/java/com/google/ads/mediation/fyber/FyberMediationAdapter.java]: There is another proposed fix for this in pull request #209

**Comment by joshuajplagonera** [comment]
Obsolete PR. Closing...

**Pull request closed without merging**